llvm/include/llvm/IR/Constants.h

```diff
@@ -975,6 +975,11 @@ class NoCFIValue final : public Constant {
     return cast<GlobalValue>(Op<0>().get());
   }
 
+  /// NoCFIValue is always a pointer.
+  inline PointerType *getType() const {
```

Comment:
```suggestion
  PointerType *getType() const {
```
We don't typically add `inline` -- unless there's a special reason here?